### PR TITLE
Add /api/clusters endpoint to vtadmin-api

### PR DESCRIPTION
**Pull request by doeg**
Signed-off-by: Sara Bee <855595+doeg@users.noreply.github.com>

## Description

Thrilling.

Response from http://localhost:14200/api/clusters, run against local Docker:

```
{"result":{"clusters":[{"id":"id1","name":"cluster1"}]},"ok":true}
```

## Related Issue(s)

N/A

## Checklist
- [ ] Should this PR be backported?
- [x] Tests were added or are not required
- [x] Documentation was added or is not required

## Impacted Areas in Vitess
Components that this PR will affect:

- [ ]  Query Serving
- [ ]  VReplication
- [ ]  Cluster Management
- [ ]  Build/CI
- [x]  VTAdmin

### CODE REVIEW

File: go/vt/vtadmin/api_test.go [hunk 1]
```diff
@@ -42,6 +42,7 @@ import (
 	"vitess.io/vitess/go/vt/vttablet/tmclient"
 
 	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+	"vitess.io/vitess/go/vt/proto/vtadmin"
```
Comment (ajm188) [inline go/vt/vtadmin/api_test.go:45]: double import here; vscode gets me with this one all the time

**Review by ajm188 — COMMENTED** [review]
one very minor thing, otherwise lg2m

File: go/vt/vtadmin/api_test.go (hunk 1 above)
Comment (doeg) [reply go/vt/vtadmin/api_test.go]: Thanks for catching that

**Review by ajm188 — APPROVED** [review]
lg2m

**Pull request merged**